test/ClangImporter/cfuncs_parse.swift

```diff
@@ -9,7 +9,7 @@ func test_cfunc1(_ i: Int) {
 }
 
 func test_cfunc2(_ i: Int) {
-  let f = cfunc2(i, 17)
+  let f = cfunc2(numericCast(i), 17)
```

Comment:
This was testing the imported type, too. I guess we're unlikely to break that at this point, though.

Reply:
Well, lets figure out how to handle the type changes in the enumeration, I suspect that will apply here too.